mesonbuild/modules/gnome.py

```diff
@@ -395,12 +395,17 @@ def generate_gir(self, state, args, kwargs):
             girtarget = girtarget.held_object
         if not isinstance(girtarget, (build.Executable, build.SharedLibrary)):
             raise MesonException('Gir target must be an executable or shared library')
+        gir_dep = kwargs.pop('gir_dep', None) or self.gir_dep
+
         try:
-            if not self.gir_dep:
+            if gir_dep == None:
```

Comment:
__[Flake8]__

[E711] comparison to None should be 'if cond is None:'


<!-- Comment by [SideCI](https://sideci.com) -->